src/main/java/redis/clients/jedis/BinaryClient.java

```diff
@@ -193,10 +193,15 @@ public void dbSize() {
     sendCommand(DBSIZE);
   }
 
+  @Deprecated
```

Comment:
Add deprecated JDoc pointing to the other methods

Reply:
?

Reply:
@gkorland Ooops!! PTAL again!